plugins/rh-rbac-backend/src/service/permission-policy.ts

```diff
@@ -27,15 +33,31 @@ p = sub, obj, act, eft
 [policy_effect]
 e = some(where (p.eft == allow)) && !some(where (p.eft == deny))
 
+[role_definition]
+g = _, _
```

Comment:
What is the reason for this change?

Reply:
I made this change, because of the bug: https://github.com/node-casbin/typeorm-adapter/issues/62 . So I provided fix for typeORM adapter: https://github.com/node-casbin/typeorm-adapter/pull/61